bugwarrior/services/gitlab.py

```diff
@@ -433,7 +431,21 @@ def include_todo(todo):
 
     def issues(self):
         tmpl = '{scheme}://{host}/api/v4/projects'
-        all_repos = self._fetch_paged(tmpl)
+
+        all_repos = []
+        if self.include_repos:
+            for repo in self.include_repos:
+                indiv_tmpl = tmpl + '/' + quote(repo, '')
+                item = self._fetch(indiv_tmpl)
+                if not item:
+                    break
```

Comment:
I see that the other invocation of `_fetch` has this `if not`/`break` clause but do we know what circumstances would cause the `item` to be falsey?

Reply:
My only thought would be if the API decided to return something that isn't a repo as normally expected. I'm happy to adjust to make the logic a little more strict than just a `falsey` check.